docs/assets/styles/main.css

```diff
@@ -31,9 +31,9 @@
   --font-size-medium: 1.65rem;
   --font-size-large: 2rem;
   --font-size-x-large: 2.5rem;
-  --font-size-xx-large: 4vw;
-  --font-size-xxx-large: 4rem;
-  --font-size-huge: 8vw;
+  --font-size-xx-large: max(22px, min(4vw, 45px));
```

Comment:
I generally like to stay away from using `px` for defining font sizes: https://andy-carter.com/blog/using-scalable-css-units-for-font-sizes

But I think it is fine here.